docker/local-dev/Dockerfile

```diff
@@ -2,11 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-FROM python:3.10-buster
+FROM python:3.8-slim
```

Comment:
Why an older Python version here? 3.8 is already EOL.

Reply:
We still support 3.8 as the lowest version in Moz-Phab https://github.com/mozilla-conduit/review/tree/main/dev/requirements, so I think it's a good way to dogfood this lowest version support.